iCub_manual/conf_manual/iCubGazeboV2_5_visuomanip/wrappers/motorControl/alljoints-mc_nws_ros2.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="alljoints-mc_nws_ros2" type="controlBoard_nws_ros2">
+    <param name="node_name"> icub_cb_node </param>
+    <param name="topic_name"> /joint_states </param>
+    <action phase="startup" level="15" type="attach">
```

Comment:
Can we use 10 and 15 here? See https://github.com/robotology/robots-configuration/issues/377 .

Reply:
Done!